src/effects/native/bessel4lvmixeqeffect.cpp

```diff
@@ -10,12 +10,13 @@ QString Bessel4LVMixEQEffect::getId() {
 EffectManifest Bessel4LVMixEQEffect::getManifest() {
     EffectManifest manifest;
     manifest.setId(getId());
-    manifest.setName(QObject::tr("Bessel4 LV-Mix EQ"));
-    manifest.setShortName(QObject::tr("Bessel4 EQ"));
+    manifest.setName(QObject::tr("Bessel4 LV-Mix ISO"));
```

Comment:
I think "Isolator" and "Equalizer" should be spelled out in the full effect names.